src/applications/vaos/new-appointment/components/ContactInfoPage.jsx

```diff
@@ -136,11 +140,17 @@ export default function ContactInfoPage() {
     bestTimeToCall: {
       'ui:title': 'What are the best times for us to call you?',
       'ui:validations':
-        flowType === FLOW_TYPES.REQUEST ? [validateBooleanGroup] : [],
+        !featureAcheronService && flowType === FLOW_TYPES.REQUEST
+          ? [validateBooleanGroup]
+          : [],
       'ui:options': {
         showFieldLabel: true,
         classNames: 'vaos-form__checkboxgroup',
-        hideIf: () => flowType === FLOW_TYPES.DIRECT,
+        hideIf: () => {
+          if (featureAcheronService)
+            return userData.facilityType === FACILITY_TYPES.VAMC;
+          return flowType === FLOW_TYPES.DIRECT;
+        },
```

Comment:
Hide best time to call checkboxes for both direct and request flows when acheron flag is true.